src/index.test.ts

```diff
@@ -680,8 +674,10 @@ describe('Running @erc725/erc725.js tests...', () => {
 
                 // Encoded value of:
                 // {
-                //   verificationFunction: 'keccak256(bytes)', // 0x8019f9b1
-                //   verificationData: '0xc41589e7559804ea4a2080dad19d876a024ccb05117835447d72ce08c1d020ec',
+                //   verification: {
+                //     method: 'keccak256(bytes)', // 0x8019f9b1
```

Comment:
Shouldn't it be 

```suggestion
                //     method: 'keccak256(utf8)', // 0x8019f9b1
```

Reply:
I think it's supposed to be the keccak of the byte stream, but you have a good point. The IPFS cid is already a hash, but I guess not for other URLs types. So keeping a hash of just the URL is probably not good verification.

Reply:
Interesting